ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -123,6 +123,11 @@ const kPartyWipeText = {
   ko: '파티 전멸',
 };
 
+const kCopiedMessage = {
+  en: 'Copied!',
```

Comment:
can you added this line, thank you.
`fr: 'Copié !',`

Reply:
Is space between é and ! intended?

Reply:
I guess it's intended looking other translations.

Reply:
it's normal in french to have this space.